### ISSUE TITLE
Add compatibility with Typer 0.6.1

### ISSUE THREAD

**Issue by lsorber** [cruft/cruft#168]
Typer v0.6.0 and v0.6.1 were recently released [1], and bring some nice features.

Cruft is currently incompatible with Typer v0.6.1:
```
Updating dependencies
Resolving dependencies...
  SolverProblemError
  Because no versions of cruft match >2.10.2,<3.0.0
   and cruft (2.10.2) depends on typer (>=0.4.0,<0.5.0), cruft (>=2.10.2,<3.0.0) requires typer (>=0.4.0,<0.5.0).
  And because typer (0.6.1) depends on typer (0.6.1)
   and no versions of typer match >0.6.1,<0.7.0, cruft (>=2.10.2,<3.0.0) is incompatible with typer (>=0.6.1,<0.7.0).
  So, because my-package depends on both typer (^0.6.1) and cruft (^2.10.2), version solving failed.
```

[1] https://github.com/tiangolo/typer/releases

**Comment by natelandau** [comment]
Ran in to this same issue.  Created https://github.com/cruft/cruft/pull/170 to solve the issue

**Comment by samj1912** [comment]
fixed in #170
